### ISSUE TITLE
After login in shows css page

### ISSUE THREAD

**Issue by shankar2004nag** [bitExpert/magento2-force-login#166]
### Preconditions

Magento Version : 2.3 C.E

Force Login Module Version : Latest

Third party modules : 

### Steps to reproduce

1.
2.
3.

### Expected result

1.

### Actual result

1. [Screenshot, logs]

**Comment by DanielRuf** [comment]
Hi @shankar2004nag,

thanks for reporting this.
Can you provide the steps to reproduce it, add a screenshot and check the console of the developer tools in your browser for bugs?

Did you also clear all caches?

**Comment by shankar2004nag** [comment]
yes, i had cleared cache.
Setup upgrade
di:compile
c:f

**Comment by DanielRuf** [comment]
I can not reproduce it using the following steps:

```shell
php72 /www/htdocs/bin/composer.phar create-project --repository-url=https://repo.magento.com/ magento/project-community-edition=2.3.2 .
find var generated vendor pub/static pub/media app/etc -type f -exec chmod g+w {} +
find var generated vendor pub/static pub/media app/etc -type d -exec chmod g+ws {} +
chmod u+x bin/magento
```

apply https://github.com/magento/magento2/issues/23405#issuecomment-511786112

```shell
php72 bin/magento setup:install ...

php72 /www/htdocs/bin/composer.phar require bitexpert/magento2-force-customer-login
php72 bin/magento module:enable BitExpert_ForceCustomerLogin

The following modules have been enabled:
- BitExpert_ForceCustomerLogin

To make sure that the enabled modules are properly registered, run 'setup:upgrade'.
Cache cleared successfully.
Generated classes cleared successfully. Please run the 'setup:di:compile' command to generate classes.
Info: Some modules might require static view files to be cleared. To do this, run 'module:enable' with the --clear-static-content option to clear them.

php72 -d memory_limit=2G bin/magento setup:di:compile
php72 bin/magento setup:upgrade
php72 bin/magento cache:flush
```

follow https://github.com/bitExpert/magento2-force-login/blob/master/docs/UserGuide.pdf

See https://magento-2-3-2.daniel-ruf.de

**Comment by shochdoerfer** [comment]
@shankar2004nag how did you install the module? Did you use the version from Marketplace or did you use Composer to install it? The version in Marketplace does not yet support 2.3, you should not be able to install if I would think.

If you follow the steps outlined by @DanielRuf the installation should work as intended.

**Comment by shankar2004nag** [comment]
I had installed the module code available in Github.

**Comment by shochdoerfer** [comment]
@shankar2004nag so you downloaded the code and placed it manually into your project? Could you describe what you have done so that we can get a better understanding? Thank you.

**Comment by shankar2004nag** [comment]
Created folder 
BitExpert / ForceCustomerLogin
Dumped all the folders and files within it.
Executed upgrade and di:compile command

**Comment by websharp** [comment]
@shochdoerfer Added a fix for this in #187
